client/src/components/create-page/CreateProfile.css

```
@@ -1,95 +1,127 @@
-  display: grid;
-  grid-template-rows: [top] 30vh [middle] 10vh [bottom] 25vh [ground] 12.5vh;
-  grid-template-columns: 100vw;
-  max-width: 100vw;
+.image-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  grid-area: 1 /1 /2/2;
+  height: 100%;
+}
+.create-profile-image {
+  border: var(--deep-blue) 3px solid;
+  height: 9em;
+  border-radius: 50%;
+  justify-self: center;
+  align-items: center;
+}
+
+.create-profile-title {
+  color: var(--white);
+  font-family: "Cookie", cursive;
+  font-size: 3em;
+  transform: rotate(-20deg) skew(-20deg);
 }
 
-.top-wrapper {
-  margin-top: 1em;
-  grid-row: top;
+.name-and-bio-wrapper {
   display: flex;
   flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  grid-area: 2 /1 /3/2;
+  height: 100%;
 }
 
-.top-wrapper-name-title {
-  width: 100%;
-  text-align: center;
-  font-family: "Cookie", cursive;
-  font-size: 3em;
+.wrapper-name-title {
+  width: 80%;
+  text-align: left;
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--white);
+  font-size: 1em;
 }
 
-.top-wrapper-name {
-  width: 30%;
-  height: 20%;
+.wrapper-bio-title {
+  width: 80%;
+  text-align: left;
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--white);
+  font-size: 1em;
+}
+
+.wrapper-name {
+  width: 80%;
+  height: 10%;
   align-self: center;
-  text-align: center;
+  text-align: left;
+  resize: none;
+  box-shadow: 0px 7px 0px 0px rgba(0, 0, 0, 0.32);
+  border-radius: 5px;
+  margin: 5px 0;
+}
+.wrapper-bio {
+  width: 80%;
+  height: 40%;
+  text-align: left;
+  resize: none;
+  box-shadow: 0px 7px 0px 0px rgba(0, 0, 0, 0.32);
+  border-radius: 5px;
+  margin: 5px 0;
 }
 
-.top-wrapper-picture-title {
+.input-file-label {
+  display: flex;
   width: 100%;
-  text-align: center;
-  font-family: "Cookie", cursive;
-  font-size: 3em;
+  justify-content: center;
 }
-
-.top-wrapper-picture {
+.browse-button {
   grid-row: bottom;
-  width: 100%;
+  width: 69px;
   height: 20%;
-  text-align: center;
+  margin: 0 1em;
+  box-shadow: 0px 7px 0px 0px rgba(0, 0, 0, 0.32);
 }
 
-.top-wrapper-upload {
+.upload-button {
+  margin: 0 1em;
   width: 30%;
   text-align: center;
-  align-self: center;
-}
-
-.middle-wrapper {
-  margin-top: 1em;
-  display: flex;
-  flex-direction: column;
-  grid-row: middle;
-  justify-self: center;
-  align-items: center;
-  height: auto;
+  box-shadow: 0px 7px 0px 0px rgba(0, 0, 0, 0.32);
 }
 
 .bio-counter {
   text-align: right;
-  width: 100%;
+  width: 80%;
   font-family: "Cookie", cursive;
   opacity: 0.5;
+  color: var(--light-blue);
 }
 
-.create-profile-image {
-  height: 90%;
-  grid-row: bottom;
-  justify-self: center;
-  align-items: center;
+.bottom-wrapper {
+  grid-area: 4 / 1 / 5 / 2;
+  display: flex;
+  width: 100%;
+  justify-content: center;
 }
 
 .create-profile-save-button {
-  grid-row: 4/5;
-  grid-column: 1/2;
-  justify-self: center;
-  align-items: center;
   font-size: 40px;
```

```suggestion
  font-size: 2.5rem;
```
better use `rem` or `em`
https://css-tricks.com/rem-global-em-local/